lines.js

```diff
@@ -14,6 +14,24 @@ var historyForward = document.querySelector(".history-button.forward");
 var currentIndex = 0;
 var isOut;
 
+historyBack.buttonAppearance = function() {
+    if (currentIndex === projectHistory.length-1) {
+        this.classList.add("activated");
```

Comment:
Work with `this` is tricky in JS. `this` can different depend on way the function is called.
For example
```
historyBack.buttonAppearance(); // `this` is  `historyBack`
```

But 
```
const buttonAppearance = historyBack.buttonAppearance;
buttonAppearance(); // `this` is  `window`
```

Or 
```
historyBack.buttonAppearance.call(null); // `this` is `null`
```

So good approach to use `this` as reference to current instance from method. That is all.